app/src/main/java/com/belatrixsf/allstars/services/CategoryServerService.java

```diff
@@ -32,7 +32,7 @@
 /**
  * Created by gyosida on 4/27/16.
  */
-public class CategoryServerService implements CategoryService {
+public class CategoryServerService extends AllStarsService implements CategoryService {
```

Comment:
I disagree to pass a "tag" to a service layer. In my opinion the service layer should not care about it, but to have the responsibility of cancelling the requests. I thought we would handle this as a set of calls per service instance, that could later be cancelled, with that approach the problem would be if we handle the ability to cancel a specific service call.